library/azure_rm_resource.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Zim Kalinowski, <zikalino@microsoft.com>
```

Comment:
isn't license under Microsoft?

Reply:
this is the pattern from all the ansible modules, there's always original author + GNU license below